core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -116,16 +110,16 @@ func (l *logPollerWrapper) LatestEvents() ([]evmRelayTypes.OracleRequest, []evmR
 	if l.proposedCoordinator != (common.Address{}) && l.activeCoordinator != l.proposedCoordinator {
 		coordinators = append(coordinators, l.proposedCoordinator)
 	}
-	nextBlock := l.nextBlock
 	latest, err := l.logPoller.LatestBlock()
 	if err != nil {
 		l.mu.Unlock()
 		return nil, nil, err
 	}
-	latest -= l.blockOffset
-	if latest >= nextBlock {
```

Comment:
What if LatestBlock() is stuck on the same number for a long time due to RPC client problems? Do we want to constantly re-query the same range?